go.mod

```diff
@@ -47,6 +47,7 @@ require (
 	github.com/aliyun/alibaba-cloud-sdk-go v1.61.1046
 	github.com/aws/amazon-ec2-instance-selector/v2 v2.0.2
 	github.com/aws/aws-sdk-go v1.38.25
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
This should not be added back, probably you are importing semver somewhere without /v4.